src/main/java/uk/gov/hmcts/cft/idam/testingsupportapi/service/TestingUserProfileService.java

```diff
@@ -166,4 +221,33 @@ protected String getEntityKey(UserProfile entity) {
     protected TestingEntityType getTestingEntityType() {
         return TestingEntityType.PROFILE;
     }
+
+    protected Set<UserProfileCategory> getUserProfileCategories(User user) {
+        return getUserProfileCategories(user.getRoleNames());
+    }
+
+    protected Set<UserProfileCategory> getUserProfileCategories(List<String> roleNames) {
```

Comment:
[minor] Probably belongs to a different class.

Reply:
yeah, I did think that. If we had an "idam library" then it could all be shared by the bridge.